### PR TITLE
Fix agones tutorial

### PR DESCRIPTION
**Pull request by scraly**

### CODE REVIEW

File: pages/platform/kubernetes-k8s/installing-agones/guide.en-asia.md [hunk 1]
```diff
@@ -170,22 +193,37 @@ The Agones <em>Hello world</em> is rather boring, a simple [Xonotic game server]
 
 Deploying a Xonotic game server over Agones is rather easy:
 
-<pre><code class="language-bash">kubectl create -f https://raw.githubusercontent.com/googleforgames/agones/release-1.27.0/examples/xonotic/gameserver.yaml</code></pre>
+```bash
+kubectl create -f https://raw.githubusercontent.com/googleforgames/agones/release-1.27.0/examples/xonotic/gameserver.yaml
+```
+
+This command install Xonotic:
```
Comment (OVH-Team-Guides) [inline pages/platform/kubernetes-k8s/installing-agones/guide.en-asia.md]:
```suggestion
This command installs Xonotic:
```

File: pages/platform/kubernetes-k8s/installing-agones/guide.en-asia.md [hunk 2]
```diff
@@ -209,57 +247,63 @@ You will see that your game server is running in a pod called `xonotic`:
...
 
+## Cleanup
+
+Uninstall Xonotic game server:
+
+```bash
+kubectl delete gameserver
+```
+
+To uninstall Agones, as you installed it through Helm, you can use `helm uninstall` command in order to delete the Agones Helm installed chart:
```
Comment (OVH-Team-Guides) [inline pages/platform/kubernetes-k8s/installing-agones/guide.en-asia.md]:
```suggestion
To uninstall Agones, as you installed it through Helm, you can use the `helm uninstall` command in order to delete the Agones Helm installed chart:
```

**Review by OVH-Team-Guides — COMMENTED** [review]
Hello @scraly, only 2 small fixes to do on all files please.
Thanks!

File: pages/platform/kubernetes-k8s/installing-agones/guide.en-asia.md (hunk 1 above)
Comment (scraly) [reply pages/platform/kubernetes-k8s/installing-agones/guide.en-asia.md]: done, thank you :)

File: pages/platform/kubernetes-k8s/installing-agones/guide.en-asia.md (hunk 2 above)
Comment (scraly) [reply pages/platform/kubernetes-k8s/installing-agones/guide.en-asia.md]: done, thank you :)

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @scraly

**Pull request merged**